stactools_cli/stactools/cli/commands/version.py

```diff
@@ -0,0 +1,14 @@
+from click import echo
+from pystac.version import get_stac_version
+from stactools.core.version import __version__
+
+
+def create_version_command(cli):
```

Comment:
Does https://click.palletsprojects.com/en/8.0.x/api/#click.version_option work? Maybe a bit simpler.

Reply:
Yeah, I _think_ this is custom enough that we have to build it ourselves.